src/transform/src/map_lifting.rs

```diff
@@ -679,13 +673,13 @@ impl LiteralLifting {
                         for key in keys.iter_mut() {
                             for expr in key.iter_mut() {
                                 #[allow(deprecated)]
```

Comment:
You forgot to remove the `allow` here.

Reply:
Good catch! I removed this one and another leftover instance in src/transform/src/dataflow.rs.